src/client.js

```diff
@@ -21,13 +22,24 @@ const viewBuildTypes = [
 const client = mozaik => {
 
     mozaik.loadApiConfig(config);
+    const caFilePath = config.get('jenkins.customCa');
+    let _certificate;
+
+    if (caFilePath.length > 0) {
+        _certificate = fs.readFileSync(caFilePath)
```

Comment:
No, you should just add the log (with the file name instead of the error message) and then throw the error